### DISCUSSION TITLE
deploying failed due to unresolved date-fns

### DISCUSSION THREAD

**Discussion by emmamalinoski** [vercel/next.js#22088] [Help]
To start off- I am a major beginner at this.  I'm trying to deploy my website on Vercel (it displays perfectly on my computer), but I keep getting a compile error because it cant resolve date-fns.  I installed it on my computer and it works fine there... any advice?  This is based on the next-js tutorial if that helps.

**Comment by krystianjj** [comment]
First check if `date-fns` is located in package.json in `dependencies` section.

**Reply by emmamalinoski** [reply]
it does not look like it is.  When i was writing the code in my text editor I thought I imported the package, tho
![Screenshot (74)](https://user-images.githubusercontent.com/78455986/107721293-4dc61780-6caa-11eb-819e-7df1af43d48d.png)

**Reply by krystianjj** [reply]
`npm install date-fns` then commit your code. It should work :)
